io/io/src/TStreamerInfoActions.cxx

```diff
@@ -963,6 +963,7 @@ namespace TStreamerInfoActions
          TVirtualCollectionProxy *proxy = fNewClass->GetCollectionProxy();
          if (proxy) {
             fCreateIterators = proxy->GetFunctionCreateIterators();
+            fCreateWriteIterators = proxy->GetFunctionCreateIterators(kFALSE);
```

Comment:
`fCreateWriteIterators` is set but seems to be unused?

Reply:
Correct ... it sole lone purpose is to remind the future implementer to use when it needs to be instead of thinking that the other one work for both read and write.